packages/web/src/pages/sign-on-page/SignOnPage.tsx

```diff
@@ -0,0 +1,162 @@
+import { useEffect } from 'react'
+
+import {
+  Box,
+  Flex,
+  IconCloseAlt,
+  Paper,
+  Text,
+  TextLink,
+  useTheme
+} from '@audius/harmony'
+import { Link, Route, Switch } from 'react-router-dom'
+import { useToggle } from 'react-use'
+
+import djBackground from 'assets/img/2-DJ-4-3.jpg'
+import djPortrait from 'assets/img/DJportrait.jpg'
+import BackgroundWaves from 'components/background-animations/BackgroundWaves'
+import { useMedia } from 'hooks/useMedia'
+import { SignInPage } from 'pages/sign-in-page/SignInPage'
+import { AudiusValues } from 'pages/sign-on-page/AudiusValues'
+import SignUpPage from 'pages/sign-up-page'
+import { SIGN_IN_PAGE, SIGN_UP_PAGE, TRENDING_PAGE } from 'utils/route'
+
+const messages = {
+  newToAudius: 'New to Audius?',
+  createAccount: 'Create an Account'
+}
+
+export const SignOnPage = () => {
+  const { isMobile } = useMedia()
+  const [isExpanded] = useToggle(false)
+  const [isLoaded, setIsLoaded] = useToggle(false)
+  const { spacing } = useTheme()
+
+  useEffect(() => {
+    setIsLoaded(true)
+  }, [setIsLoaded])
+
+  const routes = (
+    <Switch>
+      <Route path={SIGN_IN_PAGE}>
+        <SignInPage />
+      </Route>
+      <Route path={SIGN_UP_PAGE}>
+        <SignUpPage />
+      </Route>
+    </Switch>
+  )
+
+  if (isMobile) {
+    return (
+      <Flex direction='column' w='100%'>
+        <Flex
+          h={isExpanded ? '100%' : '520px'}
+          borderBottomLeftRadius={!isExpanded ? '2xl' : undefined}
+          borderBottomRightRadius={!isExpanded ? '2xl' : undefined}
+          css={{
+            position: 'absolute',
+            top: 0,
+            left: 0,
+            width: '100%',
+            backgroundColor: 'white',
+            zIndex: 1,
+            transition: '0.5s ease-in-out 0.5s',
```

Comment:
are we still waiting on harmony motion?

Reply:
yeah i talked with sammie about these values and she'd like to have them made official soon. you're right though we should at least use one of the motion tokens here.